magda-web-client/src/Components/Dataset/Add/DatasetAddCommon.ts

```diff
@@ -1234,20 +1230,59 @@ async function convertStateToDistributionRecords(
     return distributionRecords;
 }
 
+/**
+ * Update ckan export aspect status acccording to UI status.
+ * We use JSON Patch request to avoid edging cases
+ *
+ * @param {string} datasetId
+ * @param {State} state
+ */
+async function updateCkanExportStatus(datasetId: string, state: State) {
+    /**
```

Comment:
This isn't really a jsdoc comment, it should be `/* */`.

Reply:
You mean you want the `*` to be removed from 
```
```

Reply:
So a tsdoc or jsdoc is supposed to be documentation for the variable or function directly under it, like in that example. This looks like it's supposed to just be a general comment for someone reading the code (like what you'd put `//` in front of if it was on one line), rather than variable documentation for `exportDataPointer`? Or am I missing something?

Reply:
The docs software (tsdoc) won't pick up those comments as they are in the function body.
But I think those comment might make people think the comment is for `exportDataPointer` due to the comment style, as you said --- which will cause confusion.
It's now updated to use `//` now